### PR TITLE
Add contributors

### PR DESCRIPTION
**Pull request by vmcj**
I've looked at everyone since 7.0.0 tag,

### CODE REVIEW

File: doc/manual/overview.rst [hunk 1]
```diff
@@ -111,39 +112,45 @@ Marc Furon,
...
+Robert King,
 Jacob Kleerekoper,
 Jason Klein,
 Andreas Kohn,
-Yevgeny Kazakov,
-Blaz Korecic,
 Ruud Koot,
+Blaz Korecic,
 Ilya Kornakov,
+Alon Krymgand
```
Comment (eldering) [inline doc/manual/overview.rst:125]:
```suggestion
Alon Krymgand,
```

File: doc/manual/overview.rst [hunk 2]
```diff
@@ -139,6 +139,7 @@ Bert Peters,
 Ludo Pulles,
 Tobias Polzer,
 Jeroen Schot,
+Maarten Sijm,
```
Comment (eldering) [inline doc/manual/overview.rst:142]: I did not add this on purpose since he doesn't list his name publicly, but I assume @mpsijm is ok with this?

Comment (vmcj) [reply doc/manual/overview.rst:142]: Makes sense, will take it out.

Comment (mpsijm) [reply doc/manual/overview.rst]: It doesn't really matter, as my name is listed as "commit author" anyway :joy: Realising this, I might as well list my name publicly, thanks for pointing that out :slightly_smiling_face:

Comment (vmcj) [reply doc/manual/overview.rst:142]: I've added you (back) to the list.

**Review by eldering — APPROVED** [review]

**Pull request merged**